tests/Functional/Controller/ApiController80.php

```diff
@@ -497,7 +498,7 @@ public function entityWithNullableSchemaSet()
     }
 
     /**
-     * @Route("/entity-with-falsy-defaults", methods={"POST"})
+     * @Route("/entity-with-falsy-defaults", methods={"GET"})
```

Comment:
This change seems accidental and unrelated

```suggestion
     * @Route("/entity-with-falsy-defaults", methods={"POST"})
```

Reply:
Good catch! :sweat_smile: